css/03-generic.css

```diff
@@ -192,7 +192,11 @@ body.home header.entry-header {
 }
 
 /* de-hyphenate large pullquotes */
-.wp-block-pullquote .alignwide, .wp-block-pullquote .alignwide blockquote, .wp-block-pullquote .alignfull, .wp-block-pullquote .alignfull blockquote {
+.wp-block-pullquote.alignwide *, .wp-block-pullquote.alignfull * {
+     hyphens: manual;
+}
+
+.no-hyphens *{
```

Comment:
Will approve but add a space between the `*` `{` on line 199 in a future commit, please!